generic/rcutorture.py

```diff
@@ -62,37 +64,31 @@ def cpus_toggle(self):
 
         self.log.info("Online all cpus %s", totalcpus)
         for cpu in range(0, full_count):
-            online = 'echo 1 > /sys/devices/system/cpu/cpu%s/online' % cpu
-            process.system(online)
+            os.system('echo 1 > /sys/devices/system/cpu/cpu%s/online' % cpu)
         time.sleep(10)
 
         self.log.info("Offline all cpus 0 - %s\n", full_count)
         for cpu in range(0, full_count):
-            offline = 'echo 0 > /sys/devices/system/cpu/cpu%s/online' % cpu
-            process.system(offline)
+            os.system('echo 0 > /sys/devices/system/cpu/cpu%s/online' % cpu)
         time.sleep(10)
 
         self.log.info("Online all cpus 0 - %s\n", full_count)
         for cpu in range(0, full_count):
-            online = 'echo 1 > /sys/devices/system/cpu/cpu%s/online' % cpu
-            process.system(online)
+            os.system('echo 1 > /sys/devices/system/cpu/cpu%s/online' % cpu)
+        time.sleep(5)
 
         self.log.info(
             "Offline and online first half cpus %s\n", fcpu)
         for cpu in range(0, half_count):
-            offline = 'echo 0 > /sys/devices/system/cpu/cpu%s/online' % cpu
-            process.system(offline)
+            os.system('echo 0 > /sys/devices/system/cpu/cpu%s/online' % cpu)
             time.sleep(10)
-            online = 'echo 1 > /sys/devices/system/cpu/cpu%s/online' % cpu
-            process.system(online)
+            os.system('echo 1 > /sys/devices/system/cpu/cpu%s/online' % cpu)
 
         self.log.info("Offline and online second half cpus %s\n", scpu)
         for cpu in range(shalf_count, full_count):
-            offline = 'echo 0 > /sys/devices/system/cpu/cpu%s/online' % cpu
-            process.system(offline)
+            os.system('echo 0 > /sys/devices/system/cpu/cpu%s/online' % cpu)
```

Comment:
Any idea why `process.system()` did not run ? Was it because `shell=True` was not set ?